pipeline/data_preprocessing/genotype/VCF_QC.ipynb

```diff
@@ -70,16 +70,20 @@
    },
    "source": [
     "## Minimal working example\n",
+    "Following files are required to run the mwe\n",
+    "1. The target vcf file to be QC\n",
+    "2. A reference vcf.gz file to generate the dbsnp_variants\n",
+    "3. A reference fasta file\n",
+    "\n",
     "\n",
     "```\n",
-    "sos run VCF_QC.ipynb dbsnp_annotate \\\n",
-    "    --genoFile data/00-All.vcf.gz\n",
+    "nohup sos run VCF_QC.ipynb dbsnp_annotate \\\n",
+    "    --genoFile data/00-All.renamechrs.vcf.gz &\n",
     "```\n",
     "\n",
     "```\n",
-    "sos run VCF_QC.ipynb qc \\\n",
-    "    ...\n",
-    "```"
+    "nohup sos run VCF_QC.ipynb qc:3 --genoFile data/mwe_genotype.vcf --container lmm.sif  --dbsnp_variants ./00-All.renamechrs.add_chr.variants.gz --reference_genome GRCh38_full_analysis_set_plus_decoy_hla.fa -s build &\n",
```

Comment:
why `qc:3`? we are supposed to run 3 steps of `qc` from the beginning to the end. It should be `qc`